xml/hwbp_registers.xml

```diff
@@ -220,10 +220,10 @@
             size.
         </field>
         <field name="0" bits="XLEN-12:23" access="R" reset="0" />
-        <field name="sizehi" bits="22:21" access="R/W" reset="0">
-            This field only exists if XLEN is greater than 32. In that case it
-            extends \FcsrMcontrolSizelo. If it does not exist then hardware operates as if
-            the field contains 0.
+        <field name="sizehi" bits="22:21" access="WARL" reset="0">
+            This field contains the 2 low bits of the access size. The low bits
```

Comment:
Shouldn't this say "This field contains the 2 high bits of the access size..."?

Reply:
Yes. Fixed. Thanks for catching that.